plugin/src/main/groovy/com/btkelly/gnag/tasks/GnagCheckTask.java

```diff
@@ -128,36 +109,31 @@ public void taskAction() {
     }
   }
 
-  @Override
-  public Logger getLogger() {
+  private Logger getGnagLogger() {
```

Comment:
Just to clarify vs Gradle logger?

Reply:
`getLogger` is part of the `DefaultTask` class and it was causing an issue with Gradle 7 as it was not marked as an input. I tried adding the `@Internal` annotation to resolve it but it didn't so to save time I just renamed it.